src/main/java/reposense/model/CliArguments.java

```diff
@@ -9,11 +9,14 @@
  * Represents command line arguments user supplied when running the program.
  */
 public abstract class CliArguments {
+    protected Path configFolderPath;
```

Comment:
`configFolderPath` should be in ConfigCliArguments.

Reply:
Requested change has been made